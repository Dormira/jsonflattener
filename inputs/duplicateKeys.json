{
	"a": 1,
	"a": 2,
	"b": true,
	"c": {
		"c" : 4,
		"e": "true"
	}
}
