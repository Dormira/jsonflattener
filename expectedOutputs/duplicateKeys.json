{
 "a": 2,
 "b": true,
 "c.c": 4,
 "c.e": "true"
}